modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs

```diff
@@ -294,7 +294,7 @@ public class LegacyCliOptions : Dictionary<string, string>, IAgentConfigProvider
                 BatchTriggerInterval = GetValueOrDefault<TimeSpan>(LegacyCliConfigKeys.BatchTriggerInterval, TimeSpan.FromSeconds(10)),
                 MaxMessageSize = GetValueOrDefault(LegacyCliConfigKeys.MaxMessageSize, 0),
                 ScaleTestCount = GetValueOrDefault(LegacyCliConfigKeys.ScaleTestCount, 1),
-                MaxOutgressMessages = GetValueOrDefault(LegacyCliConfigKeys.MaxOutgressMessages, 200)
+                MaxEgressMessageQueue = GetValueOrDefault(LegacyCliConfigKeys.MaxEgressMessageQueue, 8192) // 8192 * 256 KB = 2 GB.
```

Comment:
>8192 [](start = 101, length = 4)

2GB ? this is all ram my docker container will get, if the service is running at 400MB avgerage memory, why collecting factor 5 the size as messages? It is very unlikley that they will ever send out. What about 1 GB as default?

Reply:
2 GB basically to account for a connectivity shortage. I wanted to use half of the available or free memory, but that info is not easy to get in .NET Core in a way that works on all platforms (would require some testing). I will change it to 1 GB as a compromise :)

Reply:
Done